### PR TITLE
Updating Turkish Json V2

### PR DESCRIPTION
**Pull request by MasterMister**
It has been exactly 3 years since the last translation, and I wanted to update it.

((If some wrongs))

### CODE REVIEW

File: VoidCases/turkish.json [hunk 1]
```diff
@@ -1,119 +1,264 @@
...
+    "settings_useinheritance": "Kullanıcı grubu mirasını kullan",
+    "sell_item": "ÖĞE SAT",
+    "settings_casesatonce": "Bir kerede en fazla oluşacak kasa sayısı",
+    "settings_currency": "Para Birimi",
+    "settings_boxcollision": "Kasa çarpışmasını devre dışı bırak",
+    "concommand": "Konsol Komutu",
+    "settings_automatic_feature": "Otomatik Öğe Öne Çıkarımı",
+    "couldnt_equip": "Donanım yapılamadı:",
+    "trading_unconfirm": "ONAYI KALDIR",
+    "amount": "MİKTAR
```
Comment (m0uka) [inline VoidCases/turkish.json]: Missing quote here, please add it

File: VoidCases/turkish.json [hunk 2]
```diff
@@ -1,119 +1,264 @@
...
+    "just_got_bought": "Ürününüz şimdi satıldı:",
+    "trading_end": "Ticaret sona erdi",
+    "hour_short": "Saat",
+    "limit_reached_perma": "Sadece %s kalıcı silah donatabilirsiniz!",
+    "trading_requestsent": "Ticaret isteği gönderildi",
+    "dont_annouce": "Duyurulmasın",
+    "annouce_win": "%s sadece %s kutusundan çıkardı!",
+    "opening_case": "KUTU AÇILIYOR: %s",
+
+    "edit_items": "Öğeleri Düzenle"
```
Comment (m0uka) [inline VoidCases/turkish.json]: Missing comma here

**Review by m0uka — CHANGES_REQUESTED** [review]
Hi, please fix these issues

File: VoidCases/turkish.json (hunk 2 above)
Comment (MasterMister) [reply VoidCases/turkish.json]: Sir, seems same with english.json line 182 space.

Comment (m0uka) [reply VoidCases/turkish.json]: The space is fine, but you are missing a comma, should be like this:
```suggestion
    "edit_items": "Öğeleri Düzenle",
```

**Comment by m0uka** [comment]
Everything else looks fine, thank you for the translation!

**Pull request merged**